contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -90,6 +90,9 @@ contract ValidatorShare is IValidatorShare {
 
     function buyVoucher(uint256 _amount) public onlyWhenUnlocked {
         uint256 share = _amount.mul(100).div(exchangeRate());
+        require(share > 0, "Insufficient amount to buy share ");
+        require(delegators[msg.sender].amount ==0, "Delegator has no rewards left");
```

Comment:
Please update the require msg to something like "Ongoing exit" or something like that.

Reply:
@AyushKaul ☝️